### PR TITLE
🌱 add build images action for release-1.6

### PR DESCRIPTION
**Pull request by tuminoid**
Image build action is needed in each release branch too.

### CODE REVIEW

**Review by lentzi90 — COMMENTED** [review]
/approve

File: .github/workflows/build-images-action.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: build-images-action
+
+on:
+  push:
+    branches:
+    - 'main'
+    - 'release-*'
```
Comment (kashifest) [inline .github/workflows/build-images-action.yml:7]: single quotes were removed in main branch, can we keep it consistent ?

Comment (tuminoid) [reply .github/workflows/build-images-action.yml:7]: I'm going to restore them back in main, if this works as expected.

**Pull request merged**